File: C++/18-二叉树的镜像.cpp
class Solution {
public:
    void Mirror(TreeNode *pRoot) {
        helper(pRoot);
    }
    void helper(TreeNode *root){
        if(!root)
            return;
        TreeNode *p = 0;
        p = root->right;
        root->right = root->left;
        root->left = p;
        helper(root->left);
        helper(root->right);
    }
};

File: Python/18-二叉树的镜像.py
class Solution:
    # 返回镜像树的根节点
    def Mirror(self, root):
        # write code here
        self.helper(root)
    def helper(self, root):
        if not root:
            return
        root.left, root.right = root.right, root.left
        self.helper(root.left)
        self.helper(root.right)
